frontend/src/types.ts

```diff
@@ -2,44 +2,59 @@
  * Common types, should be kept up to date with backend types
  */
 
-export type OdhApplication = {
-  metadata: {
-    name: string;
+export interface OdhApplication {
+  apiVersion?: string;
+  kind?: string;
+  metadata?: {
+    [k: string]: unknown;
```

Comment:
again, I don't think we need the catch alls.

Reply:
fixed using kpt's typegen which will generate the full metadata schema